### PR TITLE
Popup detail

### PR DESCRIPTION
**Pull request by BregornOriginal**
# New Feature to my portfolio: Popup added 😁

[](https://postimg.cc/5jVKw2Vh)

### Now the website has the next features:
-  The button to check project details, showup a popup with details about the project
-  When the user clicks the close (X) button, the popup disappears
- The project cards now are added dynamically whit Javascript

> Technologies used in this feature
>
> HTML, CSS & JS

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -320,6 +353,33 @@ ul {
   padding: 0;
 }
 
+/* This class were added here because Linters give me the next error
+1000:1 Expected selector ".popup-section ul li" to come before selector ".grid-section .secondary-card no-descending-specificity
+li"
+ 1050:1 Expected selector ".text-container ul li" to come before selector ".grid-section .secondary-card no-descending-specificity
+li" */
+
+.popup-section ul li {
```
Comment (Ismailco) [inline main.css:362]: - [ ] You may consider removing the space between the ```See Live``` and ```See source``` buttons so your design is as close as possible to the Figma design

Screenshot_20220512_210131

File: main.css [hunk 2]
```diff
@@ -19,6 +19,31 @@ ul {
...
+}
+
+/* blur is for header and main for popup */
+.blur {
+  filter: blur(0.8rem);
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+/* disable-scroll for body for popup */
+.disable-scroll {
```
Comment (Ismailco) [inline main.css:33]: - [ ] On mobile the popup is not completed and can't scroll down, kindly fix this or remove the ```.disable-scroll``` so the user can reach the popup links

Screenshot_20220512_210557

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @BregornOriginal,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Follow best practices
✔️ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] You have some typos in the README, please consider fixing them

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @BregornOriginal , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**